plugins/woocommerce/tests/legacy/unit-tests/rest-api/Helpers/ProductHelper.php

```diff
@@ -24,10 +24,11 @@ class ProductHelper {
 
 	/**
 	 * Counter to insert unique SKU for concurrent tests.
+	 * The starting value ensures no conflicts between existing generators.
 	 *
 	 * @var int $sku_counter
 	 */
-	private static $sku_counter = 0;
+	private static $sku_counter = 100000;
```

Comment:
Here and other test files: fixes failing tests in main suite surfaced after sharding introduction.